src/useMutation.ts

```diff
@@ -21,6 +24,7 @@ export type MutationUpdaterFn<TData = Record<string, any>> = (
 export interface BaseMutationHookOptions<TData, TVariables>
   extends Omit<MutationOptions<TData, TVariables>, 'mutation' | 'update'> {
   update?: MutationUpdaterFn<TData>;
+  rethrow?: boolean;
```

Comment:
Would it be possible to add callbacks to `onCompleted` and `onError` like the react-apollo's Mutation component?